src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/Settings.Cache/SettingsCacheTest.cs

```diff
@@ -38,74 +34,70 @@ public class SettingsCacheTest
         [TestMethod]
         public void GetSettings_SettingNotInCache_SettingsReadFromFile()
         {
-            var issues = CreateIssues();
```

Comment:
Fixed up the tests to use instances of the new wrapper, rather that lists of issues.